src/3rd_party_bundle_add.c

```diff
@@ -21,69 +21,6 @@
 
 #include "swupd.h"
```

Comment:
Same here, wrap around #ifdef THIRDPARTY

Reply:
It's correct to include that after the ifdef. The config.h is included by swupd.h and it contains the compile flag that the ifdef uses.